ChangeLog.d/8726.txt

```diff
@@ -0,0 +1,3 @@
+Features
+   * Add platform support for z/OS.
+   
```

Comment:
The CI is complaining about trailing whitespaces in the empty line. To be safe I suggest to delete the whole line. Also, if we are already changing it, we could indicate that the support we added is not complete.

```suggestion
Features
   * Add partial platform support for z/OS.
```

Reply:
Thank you, that's a good idea indeed.